utils/get_prometheus_data.py

```diff
@@ -0,0 +1,145 @@
+import os
+import requests
+import json
+import logging
+import urllib3
+from datetime import datetime, timedelta
+import time
+import sys
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+logger = logging.getLogger("snafu")
+
+class get_prometheus_data():
+    def __init__(self, action):
+
+        self.uuid = action["uuid"]
+        self.user = action["user"]
+        self.cluster_name = action["cluster_name"]
+        self.test_config = action["test_config"]
+        self.start = action["starttime"]
+
+        # change datetime in seconds string to datetime object
+        endtime_object = datetime.fromtimestamp(int(action["endtime"]))
+        # add 120s buffer to end time
+        endtime_object = endtime_object + timedelta(seconds=120)
+        # convert back to string
+        self.end = endtime_object.strftime('%s')
+
+        # step value to be used in prometheus query
+        self.T_Delta = 30
```

Comment:
why is this hard-coded?  Are there any circumstances when you would want to change it?

Reply:
First attempt was to hard code it to 30s, there are circumstances that users may want to change it but didn't want to cover than in the initial PR.